packages/react-composites/src/composites/CallComposite/adapter/CallAdapter.ts

```diff
@@ -121,6 +121,11 @@ export type CallAdapterClientState = {
    * Default set of background images for background replacement effect.
    */
   videoBackgroundImages?: VideoBackgroundImage[];
+  /* @conditional-compile-remove(video-background-effects) */
+  /**
+   * Default set of background images for background replacement effect.
```

Comment:
Does this also include the custom backgrounds that contoso would upload or provide to their composite?